steps/appaloosa/1.0.0/step.yml

```diff
@@ -0,0 +1,167 @@
+title: Publish to Appaloosa Store
+summary: This step allows you to publish your application to your private app store
+  on Appaloosa Store
+description: |
+  Bitrise Step to Deploy an iOS IPA to Appaloosa-Store.
+
+  Required: an Xcode Archive Step should run before this Step!
+
+  This step requires an Appaloosa-Store account or an email address to create your account
+website: https://www.appaloosa-store.com
+source_code_url: ""
+support_url: ""
+published_at: 2015-10-23T14:36:17.786395069+02:00
+source:
+  git: https://github.com/appaloosa-store/bitrise_appalossa.git
+  commit: 5e0095698cc0120fe6fba4298c0cd334c127ae15
+host_os_tags:
+- osx-10.9
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- deploy
+- appaloosa
+- appaloosa-store
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- BITRISE_IPA_PATH: $BITRISE_IPA_PATH
+  opts:
+    description: This is the variable of your IPA path on Bitrise
+    is_dont_change_value: false
+    is_expand: true
+    is_required: true
+    summary: ""
+    title: IPA PATH
+- USER_EMAIL: null
+  opts:
+    description: |-
+      ***Only if you haven't created an account on Appaloosa yet.***
+
+      You'll receive an email with your credentials to access your private store.
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: If you don't have an account we'll automatically create one for you.
+      At the end of the step you'll receive an email with your credentials to access
+      your store, and install your application. For all future runs of this step,
+      remove your email address and setup this step as a registered user.
+    title: User Email Address
+- APPALOOSA_API_KEY: null
+  opts:
+    description: '***Only if you are already registered on Appaloosa***'
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: This is your unique Appaloosa API Key that can be found in your user's
+      profile on Appaloosa
+    title: User API Key
+- STORE_ID: null
+  opts:
+    description: |-
+      ***Only if you are already registered on Appaloosa***
+
+      This is the store id of the store you want your application uploaded to.
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: You can find this value in your store settings
+    title: Store ID
+- SCREENSHOT1: null
```

Comment:
Is it expected that a user will upload the screenshots to another site, and provide the URLs for this step? What's the purpose of this field, if the screenshot is not generated by the build? The user will still upload the screenshots manually, right?